app/controllers/users_controller.rb

```diff
@@ -1,4 +1,27 @@
 class UsersController < ApplicationController
+
+  def show
+    @user = User.find(params[:id])
+  end
+
   def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render 'new', status: :unprocessable_entity
+    end
   end
+
+  private
+
+    def user_params
+      params.require(:user).permit(:name, :email, :password,
+                                   :password_confirmation)
+    end
```

Comment:
```suggestion
  private

  def user_params
    params.require(:user).permit(:name, :email, :password, :password_confirmation)
  end
```

private メソッド内でのメソッド定義はインデントを private 行に揃えるのが慣習です